T3/merging/merge.py

```diff
@@ -30,7 +30,7 @@
     from PandaCore.Tools.process import *
     xsecscale = 1
 
-sys.path.append('configs/')
+sys.path.append('PandaAnalysis/T3/merging/configs/')
```

Comment:
No. This is path-dependent, depending on where you call it from. We can make it path-independent, but what you propose requires you call it from `$CMSSW_BASE/src`

Reply:
Yeah, that's true. How could we run it without going into PandaAnalysis/T3/merging? Any suggestion is welcome

Reply:
How about `os.getenv('CMSSW_BASE') + '/src/PandaAnaysis/T3/merging/configs'` so there is an explicit path?

Reply:
Yes, that's perfect!